Frequently_used_code/ARD_overpass_predictor.ipynb

```diff
@@ -0,0 +1,1677 @@
+{
```

Comment:
<p>Why are the secondary overpasses not already in datetime format? I think you can get <code>pd.read_csv</code>  to force them all into datetime format automatically.</p><p> </p><p>What is a secondary overpass and why would I want to have one in my input? Could you please add a little explanation?</p><p> </p><p>Don't use <code>os.chdir</code> as it has a tendency to make the rest of the notebook harder to understand and may break existing scripts.  Read the file using a relative path instead. You also can't assume that this notebook is in <code>jovyan/</code> (e.g. I am testing it in a different place!) so try something like <code>../Supplementary_data/ARD_overpass_predictor/overpass_input.csv</code>  instead.</p><p> </p><p>Input file looks good!</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GeoscienceAustralia/dea-notebooks/pull/736/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>